### DISCUSSION TITLE
What is the "speed" attribute in spaCy's evaluate?

### DISCUSSION THREAD

**Discussion by vmatter** [explosion/spaCy#10913] [Help: Other Questions] [answered]
I would like to know if the `SPEED` attribute in spaCy's evaluate function is the prediction time, running time, or something else...

![speed](https://user-images.githubusercontent.com/43481916/172019026-99cab327-70d7-4d89-adb6-4f66037608b1.png)

Many thanks!

**Comment by polm** [comment, marked as answer]
It's words processed per second. A little buried in the docs but you can find it in the "Understanding the training output and score types" dropdown here.

https://spacy.io/usage/training/#score-types

**Reply by vmatter** [reply]
Many thanks, I was not able to find it on my own... 😅

**Reply by polm** [reply]
No worries, it is kind of hard to find.
